demo/README.md

```diff
@@ -8,24 +8,49 @@ The following will provide a Tekton pipeline that can build an example python ap
 
 The details of each build type and the steps required to support Pelorus can be found in the [tekton manifest](./tekton-demo-setup/03-build-and-deploy.yaml)
 
-Example:
+
+Instructions:
+
+* Fork the konveyor/pelorus git repository to your github organization.
+
+* Add or configure your github-secret with your Github token
+
 ```
-./demo-tekton.sh -g https://github.com/konveyor/pelorus.git -b binary
+oc create secret generic github-secret --from-literal=TOKEN=ghp_<snip> -n pelorus
+```
+
+* Copy the sample Pelorus instance manifest in pelorus/demo/operator_tekton_demo_values.yaml.sample
```

Comment:
the path is wrong? missing `tekton-demo-setup` folder?

I would put the path between `code snipet` and remove `pelorus/demo/` because on the tekton script command we assume the user is in the demo folder (maybe add this as the first sentence)

Reply:
ugh